wire-tap/wire-tap.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+
+type SnowflakeRequest record {
+    string statement;
+    string timeout = "60";
+    string database = "messageLog";
+    string schema = "message";
+    string role = "logger";
+};
+
+type WireTapRequest record {|
+    string 'table;
+    "INFO"|"WARNING"|"ERROR" severity;
+    string message;
+|};
+
+type StockResponse record {
+    string parentHandlingUnitUUID;
+    string StockItemUUID;
```

Comment:
inconsistency with the naming conventions.

Reply:
Will have to use the names used in the json response here